src/cli/tc.rs

```diff
@@ -31,9 +33,60 @@ pub async fn run() -> Result<()> {
         Err(e) => Err(e),
       }
     }
+    Command::Init { file_path } => Ok(init(&file_path).await?),
   }
 }
 
+pub async fn init(file_path: &str) -> Result<()> {
+  let tailcallrc = fs::read_to_string("assets/.tailcallrc.graphql")?;
```

Comment:
This will only work if the file exists at this path. Once the release is made, this file won't be available on the user's machine. I would recommend using https://crates.io/crates/resource to embed the configuration inside the binary at compile time.